### DISCUSSION TITLE
pitch: rename contracts / variables

### DISCUSSION THREAD

**Discussion by thelostone-mc** [allo-protocol/aips#21] [New Ideas]
### Description

**Contract Rename**
- ProjectRegistry.sol to be renamed as Registry.sol
- VotingStrategy renamed as AllocationStrategy.sol
- PayoutStrategy renamed as Distribution Strategy 

- Additionally variable renaming based on contract changes

**Comment by 0xKurt** [comment]
We totally should rename the contracts, especially the `VotingStrategy` and the `vote()` function since the names are just misleading.

+1

**Reply by nfrgosselin** [reply]
I agree with renaming, as well as renaming `VotingStrategy` and `vote()` as @KurtMerbeth called out. 

I also want to rename the Round concept (i.e. `IRoundImplementation`, `RoundFactory`, etc) to something more generic. My preference would be to call that core contract something like `Pool` or `Fund`, but it would be nice to get some other opinions in here. 

@thelostone-mc does this feel like an urgent change? or something that we could do more holistically?

**Reply by 0xKurt** [reply]
at least the vote strategy would be a nice cosmetic change for the DirectGrants. Since they are not voting at all. 
if we would introduce `allocate()` instead it just would make much more sense.

I don't have a better name for `Round` (just Manager :D), but using `Pool` or `Fund` are misleading too, or?
i expect a Pool or Fund to pool money in a way.

**Comment by codenamejason** [comment]
Adding some implementation details here: #35
